tests/unit/gist/controller-test.js

```diff
@@ -0,0 +1,42 @@
+import Ember from "ember";
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('controller:gist', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+  needs: ['service:ember-cli']
+});
+
+// Replace this with your real tests.
+test('it exists', function(assert) {
+  var controller = this.subject();
+  assert.ok(controller);
+});
+
+test('deleting a gist requires confirmation', function(assert) {
+  let cacheConfirm = window.confirm;
```

Comment:
We should grab this in the `beforeEach` and reset it in the `afterEach`.  The main reason is that if an error is thrown during the test `window.confirm` will not be reset (and will likely cause other tests to fail in the future).

Something like:

```javascript
moduleFor('controller:gist', {
  needs: ['service:ember-cli'],
  beforeEach() {
    this._originalConfirm = window.confirm;
  },

  afterEach() {
    window.confirm = this._originalConfirm;
  }
});
```